src/lambdas/api/app.js

```diff
@@ -129,17 +129,10 @@ app.get('/collections/:collectionId/items', async (req, res, next) => {
 })
 
 app.post('/collections/:collectionId/items', async (req, res, next) => {
-  try {
-    res.json(await api.searchItems(
-      req.params.collectionId,
-      req.body,
-      satlib.es,
-      req.endpoint,
-      'POST'
-    ))
-  } catch (error) {
-    next(error)
-  }
+  if (!process.env['ENABLE_TRANSACTIONS_EXTENSION']) next(createError(404))
+
+  // todo: implement
+  next(createError(501))
```

Comment:
Because the previous `next()` call was a return, you'll want to return this one as well for return-type consistency.

```suggestion
  return next(createError(501))
```

Reply:
when are you supposed to return next and when are you supposed to just call it?

Reply:
nm, I see the warning about consistent return types now, I just wasn't thinking about it right